index.html

```diff
@@ -1,13 +1,45 @@
 <!DOCTYPE html>
-<html lang="en-US">
+<html>
     <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>The name of your website</title> 
+        <title>What Stand Are You?</title> 
         <link rel="stylesheet" href="styles.css">
         <script type="text/javascript" src="script.js"></script>
     </head>
     <body>
-        Your HTML goes here
+        <h1>What is YOUR Stand?</h1>
```

Comment:
Nice job using a <select> tag with specific values different from the visible ones. Makes the quiz feel more genuine